### ISSUE TITLE
Ubuntu 20.04 LTS

### ISSUE THREAD

**Issue by hebasto** [bitcoin/bitcoin#18419]
Ubuntu 20.04 LTS (Focal Fossa) is going to be [released](https://wiki.ubuntu.com/FocalFossa%20/ReleaseSchedule) on April 23. Users will run the upcoming 0.20 version on it too. So I decided to test `focal`.

System packages:
- GNOME 3.36
- `g++` 9.3.0
- `boost` 1.71
- `Qt` 5.12.5 LTS
- `libevent` 2.1.11

On `master` (ac579ada7e83a1b8100d611412f9ede885a4e522) configured:
```
$ ./autogen.sh
$ ./configure --with-incompatible-bdb --disable-ccache
```

The following `make` emits massive Qt related `-Wdeprecated-copy` warnings like this:
```
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qmetaobject.h:45,
                 from /usr/include/x86_64-linux-gnu/qt5/QtTest/qtestcase.h:48,
                 from /usr/include/x86_64-linux-gnu/qt5/QtTest/qtest.h:45,
                 from /usr/include/x86_64-linux-gnu/qt5/QtTest/QTest:1,
                 from ./qt/test/compattests.h:9,
                 from qt/test/compattests.cpp:9:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvariant.h: In constructor ‘QVariant::QVariant(QVariant&&)’:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvariant.h:275:25: warning: implicitly-declared ‘QVariant::Private& QVariant::Private::operator=(const QVariant::Private&)’ is deprecated [-Wdeprecated-copy]
  275 |     { other.d = Private(); }
      |                         ^
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvariant.h:401:16: note: because ‘QVariant::Private’ has user-provided ‘QVariant::Private::Private(const QVariant::Private&)’
  401 |         inline Private(const Private &other) Q_DECL_NOTHROW
      |                ^~~~~~~
```

This issue was previously reported and discussed in #15822.

---

The application menu icon looks like this (https://github.com/bitcoin/bitcoin/issues/18419#issuecomment-617802329):

![Screenshot from 2020-03-24 17-29-26](https://user-images.githubusercontent.com/32963518/77444977-2dbd0980-6df5-11ea-825d-37d0d74eef7b.png)

---

The "Show / Hide" item in the trayicon menu does not work:

![DeepinScreenshot_select-area_20200324173505](https://user-images.githubusercontent.com/32963518/77445488-d5d2d280-6df5-11ea-9ac1-c7bb2fc536b1.png)

---

~Clicking the "Window -> Minimize" menu item disables it but does not minimize the main window:~

![Screenshot from 2020-03-24 17-43-56](https://user-images.githubusercontent.com/32963518/77446421-11ba6780-6df7-11ea-99fa-62a343c5b888.png)

~But the "Ctrl+M" shortcut works as expected.~ Fixed in #18549.

**Comment by hebasto** [comment]
> Clicking the "Window -> Minimize" menu item disables it but does not minimize the main window

Could be fixed by #18549.

**Comment by hebasto** [comment]
UPDATE:
```
$ qmake -v
QMake version 3.1
Using Qt version 5.12.8 in /usr/lib/x86_64-linux-gnu
```

... but Qt code warnings are still emitted by both GCC 9.3 and Clang 10.0 compilers.

**Comment by hebasto** [comment]
The distorted icon is an [upstream bug](https://discourse.ubuntu.com/t/ubuntu-20-04-beta-bugs-and-current-state/15259).

**Comment by hebasto** [comment]
> The following `make` emits massive Qt related `-Wdeprecated-copy` warnings

Fixed by #18738.

**Comment by hebasto** [comment]
> The "Show / Hide" item in the trayicon menu does not work:

This bug has been separated to #19186.
